api/rest.go

```diff
@@ -0,0 +1,40 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/TheChaseBL/pokerepo-backend/database"
+	"github.com/TheChaseBL/pokerepo-backend/entity"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+)
+
+func AddPokemonHandler(w http.ResponseWriter, r *http.Request) {
+	var p entity.Pokemon
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&p)
+	if err != nil {
+		panic(err)
```

Comment:
I suggest we be consistent and `log.Fatal(err)`.

Reply:
Yeah, actually what I/we need to do is send an error response to the requestor in this particular case as well.